docs/release.md

```diff
@@ -0,0 +1,149 @@
+# steps
+
+1. Sync latest master
+1. new branch
+1. Sync the downstream pipeline repo
+1. Checkout release branch
+1. Copy the release yaml from the pipelines repo to operator
+   deploy/resources/<version>
+1. modify operator config_controller.go to update the <version>
+1.  test the operator using `up local`
```

Comment:
let's add command here, otherwise the `--namespace ""` flag might be missed

`operator-sdk up local --namespace ""`